cdap-data-fabric/src/test/java/co/cask/cdap/data2/datafabric/dataset/DatasetsUtilTest.java

```diff
@@ -57,9 +58,14 @@ public void testFixProperties() throws DatasetManagementException, UnsupportedTy
             FileSetProperties.builder().setEnableExploreOnCreate(true).setExploreFormat("csv").build());
 
     testFix("timePartitionedFileSet",
-            FileSetProperties.builder().setBasePath("relative").build());
+            FileSetProperties.builder()
+              .setBasePath("relative")
+              .add(PartitionedFileSetDefinition.NAME_AS_BASE_PATH_DEFAULT, "true").build());
```

Comment:
hmmm - because this sets a base path, the NAME_AS_BASE_PATH should not apply and not be needed, right?

Reply:
ah you're right. This is leftover from when I had implemented compatibility incorrectly, will remove.